lib/srv/desktop/rdp/rdpclient/client.go

```diff
@@ -108,17 +108,33 @@ func init() {
 }
 
 // Client is the RDP client.
+// It's lifecycle is:
+//
+// ```
+// rdpc := New() // creates client and kicks off RDP connection
+// rdpc.Wait()   // waits for the duration of the connection
+// ```
+//
+// rdpc.Wait() typically takes care of calling rdpc.Cleanup().
+// However, if something fails between the call to New() and
+// rdpc.Wait(), the caller MUST call rdpc.Cleanup() to ensure
+// all memory is released.
```

Comment:
This is a confusing API and I would like to avoid it.

How is someone supposed to know if "something fails between New() and Wait()" anyway?

Is there some way we could perhaps:
- unexport `cleanup()`
- make it a required convention to always call `Wait()` after new, to ensure cleanup happens
- make `Wait()` just perform cleanup and return immediately if something failed with `New`